### DISCUSSION TITLE
dot accessor on Collection object is not working

### DISCUSSION THREAD

**Discussion by patrick-leb** [livewire/livewire#5076] [1. Help]
Hi there,
I had a bit of a head scratcher on this issue. I believe, since Collection objects are supported ,(according to the docs) that the dot accessor should function like it does for the array. 
The temporary fix is to use `->toArray()` before assigning to the property.

## Component
<img width="401" alt="image" src="https://user-images.githubusercontent.com/68301566/176273235-cd930160-8649-413a-a10a-ddb02c003746.png">

## View
<img width="519" alt="image" src="https://user-images.githubusercontent.com/68301566/176273329-31a983fc-ba7f-4e95-b1ae-b975c4876075.png">

## Result
<img width="486" alt="image" src="https://user-images.githubusercontent.com/68301566/176273400-de644b51-b5df-40ce-8993-07dc110dc99c.png">

## Expectation
<img width="465" alt="image" src="https://user-images.githubusercontent.com/68301566/176273702-2a68010a-7a04-424a-a408-36b4a6756886.png">

```
livewire:v2.10.5
Laravel:v9.17.0
php:8.0
Safari:15.1
```

**Comment by joshhanley** [comment]
@patrick-leb hmm that has worked for me in the past. Can you share a component class and view as text rather than screenshot so we can have a look? Thanks!
